packages/pilot/src/containers/Filter/Tags/style.css

```
@@ -0,0 +1,17 @@
+@import "former-kit-skin-pagarme/dist/styles/spacing.css";
+
+.selectedOptionsTitle {
+  font-size: 14px;
+  text-transform: uppercase;
+  color: #a0a0a0;
```

E esta cor aqui deveria ser 
```suggestion
color: var(--color-light-steel-100);
```

---

Alterado